Src/main.c

```diff
@@ -409,6 +443,10 @@ static void SystemClock_Config (void)
         Error_Handler();
     }
 
+  #if defined(BOARD_FLEXI_HAL)
+    __HAL_RCC_USB_OTG_FS_CLK_ENABLE();
+  #endif
+
```

Comment:
And why this for FlexiHAL only? It is called from `USBD_LL_Init()` during USB initialisation.